spec/rspec/matchers/english_phrasing_spec.rb

```diff
@@ -0,0 +1,76 @@
+module RSpec
+  module Matchers
+    RSpec.describe EnglishPhrasing do
+      describe ".split_words" do
+        it "replaces underscores with spaces" do
+          expect(described_class.split_words(:banana_creme_pie)).to \
+            eq("banana creme pie")
+        end
+
+        it "first casts its argument to string" do
+          x = double
+          allow(x).to receive(:to_s).and_return("banana")
+          expect(described_class.split_words(x)).to eq("banana")
+        end
+      end
+
+      describe ".list" do
+        context "given nil" do
+          it "returns value from inspect, and a leading space" do
+            expect(described_class.list(nil)).to eq(" nil")
+          end
+        end
+
+        context "given a Struct" do
+          it "returns value from inspect, and a leading space" do
+            banana = Struct.new("Banana", :flavor)
+            allow(banana).to receive(:inspect).and_return("Delicious")
+            expect(described_class.list(banana)).to eq(" Delicious")
+          end
+        end
+
+        context "given an Enumerable" do
+          before do
+            allow(described_class).to \
+              receive(:item_description).and_return("Banana")
+          end
+
+          context "with zero items" do
+            it "returns the empty string" do
+              expect(described_class.list([])).to eq("")
+            end
+          end
+
+          context "with one item" do
+            let(:list) { [double] }
+            it "returns description, and a leading space" do
+              expect(described_class.list(list)).to eq(" Banana")
+              expect(described_class).to \
```

Comment:
I'm really not a fan of the use of `\` between `to` and the matcher.  It's not a style we use anywhere else in RSpec's specs. You mind putting this all on one line?  If you feel some of these are two long, you can use the styles we use elsewhere:

``` ruby
expect(
  some_long_expression.that_goes_on_its_own_line
).to something

# or

expect(some_expression).to matcher(
  matcher_args,
  can, go,
  on, other_lines
)
```

(Or some combination thereof).